packages/terra-form-textarea/src/Textarea.jsx

```diff
@@ -188,7 +195,10 @@ class Textarea extends React.Component {
 
     return (
       <textarea
-        ref={(textarea) => { this.textarea = textarea; }}
+        ref={(textarea) => {
+          this.textarea = textarea;
```

Comment:
This is probably frowned upon, but I wanted to mention that you already have access to this textarea ref from the component ref. This is a part of the public API.

```
<Textarea ref={(ref) => this.componentRef = ref} />
```
```
console.log(this.componentRef) // Class reference
```

```
console.log(this.componentRef.textarea); // Dom node
```

Reply:
I learned something new today. Is this PR still desired? I'd be happy to close it, looks like this solves our needs.

Reply:
I don't know if it's appropriate to access variables this way, but it does work. 

It might be worth doing a little investigation and discuss if this is considered an implementation leak or a valid way to access underlying references. 

It gets a little blurry when accessing an exposed / public internal variable.

Reply:
I suspect this is probably an anti-pattern, unless we expect the private `textarea` variable name to persist. Although it's somewhat "cleaner" in terms of implementation, it feels wrong to expect a private variable to be maintained as a public API contract.

Reply:
As far as I know, we've only ever considered passivity within the context of propTypes and module exports. The second we expose the entire instance itself as an API, we put ourselves in a really tricky position long-term.